### ISSUE TITLE
VTK port link to Release dependencies in Debug configuration

### ISSUE THREAD

**Issue by KindDragon** [Microsoft/vcpkg#1419]
expat.lib, zlib.lib and others
```
1> CMake Warning at D:/Work/vcpkg/scripts/buildsystems/vcpkg.cmake:113 (_add_executable):
1>   Cannot generate a safe linker search path for target foot_measurements
1>   because there is a cycle in the constraint graph:
1> 
1>     dir 0 is [D:/Work/.CMakeBuild/9e97eb63-7cf7-c83d-bf8f-e2b1358b01a8/build/x64-vc15-Debug/lib/Release]
1>     dir 1 is [D:/Work/vcpkg/installed/x64-windows/lib]
1>       dir 2 must precede it due to link library [OpenNI2.lib]
1>     dir 2 is [D:/Work/vcpkg/installed/x64-windows/debug/lib]
1>       dir 1 must precede it due to link library [expat.lib]
1>     dir 3 is [C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v8.0/lib/x64]
```

**Comment by albertziegenhagel** [comment]
This should be fixed by the PR #1601

**Comment by alexkaratarakis** [comment]
#1601 is now merged, let me know if that resolves the issue.
